operator/cluster-operator.go

```diff
@@ -0,0 +1,135 @@
+package operator
+
+import (
+	crv1 "github.com/grtl/mysql-operator/pkg/apis/cr/v1"
+	v1beta2 "k8s.io/api/apps/v1beta2"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const MySQLPortNumber = 3306
+
+func AddCluster(cluster *crv1.MySQLCluster, kClientset kubernetes.Interface) {
+	createServiceForCluster(cluster, kClientset)
+	createStatefulSetForCluster(cluster, kClientset)
+}
+
+func createServiceForCluster(cluster *crv1.MySQLCluster, kClientset kubernetes.Interface) {
+	servicesInterface := kClientset.CoreV1().Services(cluster.ObjectMeta.Namespace)
+
+	newService := serviceForCluster(cluster)
+	_, err := servicesInterface.Create(&newService)
+
+	if err != nil && !apierrors.IsAlreadyExists(err) {
```

Comment:
To be consistent let's not put an empty line between function call and the result check.